dateparser/languages/locale.py

```diff
@@ -207,6 +207,10 @@ def translate_search(self, search_string, settings=None):
                 elif self._token_with_digits_is_ok(word):
                     translated_chunk.append(word)
                     original_chunk.append(original_tokens[i])
+                # Use original token because word_is_tz is case sensitive
+                elif translated_chunk and word_is_tz(original_tokens[i]):
+                    translated_chunk.append(word)
+                    original_chunk.append(original_tokens[i])
                 else:
                     if translated_chunk:
```

Comment:
:lipstick: I’m thinking maybe we could replace

```
elif translated_chunk and word_is_tz(original_tokens[i]):
    # 1
else:
    if translated_chunk:
        # 2
```

with

```
elif translated_chunk:
    if word_is_tz(original_tokens[i]):
        # 1
    else:
        # 2
```

Reply:
I prefer to keep the current way as I think the logic is more clear with such case conditions:
```
if word-is-in-dictionary:
  # 1
elif word-is-digit:
  # 2
elif word-is-timezone:
  #3 
else:
  if has-chunk:
    # 4
```